locale/pt-br/texts.po

```diff
@@ -21,25 +79,338 @@ msgstr ""
+#: src/screens/CreateTokenConfirm.js:135
+#, javascript-format
+msgid "Amount of ${ this.name }"
+msgstr "Quantidade de ${ this.name }"
+
+#: src/screens/CreateTokenConfirm.js:143
+msgid "Token name"
+msgstr "Nome do token"
+
+#: src/screens/CreateTokenConfirm.js:149 src/screens/CreateTokenSymbol.js:70
+msgid "Token symbol"
+msgstr "Símbolo do token"
+
+#: src/screens/CreateTokenConfirm.js:155
+msgid "Deposit"
+msgstr "Depósito"
+
+#: src/screens/CreateTokenConfirm.js:162
+msgid "Create token"
+msgstr "Criar um token"
+
+#: src/screens/CreateTokenDepositNotice.js:45
+msgid "When creating new tokens, a "
+msgstr "Quando um novo token é criado, um "
+
+#: src/screens/CreateTokenDepositNotice.js:47
+#, javascript-format
+msgid "deposit of ${ depositPercentage }% "
+msgstr "depósito de ${ depositPercentage }% "
+
+#: src/screens/CreateTokenDepositNotice.js:49
+msgid ""
+"in HTR is required - e.g. if you create 1000 NewCoins, 10 HTR are needed as "
+"deposit."
+msgstr ""
+"é exigido em HTR - por exemplo, se você criar 1.000 MinhaMoeda, 10 HTR serão "
+"exigidos como depósito."
+
+#: src/screens/CreateTokenDepositNotice.js:52
+msgid ""
+"If these tokens are later melted, the HTR deposit will be returned. Read "
+"more about it "
+msgstr ""
+"Se esses tokens forem destruídos futuramente, o depósito em HTR será "
+"devoldido. Leia mais sobre esse depósito "
+
+#: src/screens/CreateTokenDepositNotice.js:53
+msgid "here"
+msgstr "aqui"
+
+#: src/screens/CreateTokenDepositNotice.js:60
+msgid "I understand"
+msgstr "Eu concordo"
+
+#: src/screens/CreateTokenDetail.js:42 src/screens/TokenDetail.js:41
+msgid "TOKEN DETAILS"
+msgstr "DETALHES DO TOKEN"
+
+#: src/screens/CreateTokenName.js:27
+msgid "0/30 characters"
+msgstr "0/30 caracteres"
+
+#: src/screens/CreateTokenName.js:33
+#, javascript-format
+msgid "${ text.length }/30 characters"
+msgstr "${ text.length }/30 caracteres"
+
+#: src/screens/CreateTokenName.js:60
+msgid "Token name should be the full name of the new token you are creating"
+msgstr ""
+"O nome do token deve ser o nome completo do novo token que você está criando"
+
+#: src/screens/CreateTokenName.js:61
+msgid "E.g. MyToken or My Token"
+msgstr "Por exemplo, MeuToken ou Meu Token"
+
+#: src/screens/CreateTokenSymbol.js:73
+msgid "Between 2 and 5 characters"
+msgstr "Entre 2 e 5 caracteres"
+
+#: src/screens/CreateTokenSymbol.js:80
+msgid "The symbol is a smaller version of the token name"
+msgstr "O símbolo é a versão reduzida do seu token."
+
+#: src/screens/CreateTokenSymbol.js:81
+msgid "E.g. HTR"
+msgstr "Por exemplo, HTR"
+
+#: src/screens/Dashboard.js:152 src/screens/RegisterTokenManual.js:133
+msgid "Register token"
+msgstr "Registrar um token"
+
+#: src/screens/Dashboard.js:159
+msgid "TOKENS"
+msgstr "TOKENS"
+
+#: src/screens/InitWallet.js:57
+msgid "Welcome to Hathor Wallet!"
+msgstr "Bem vindo à Hathor Wallet"
+
+#: src/screens/InitWallet.js:60
+msgid "This wallet is connected to the"
+msgstr "Esta carteira está conectada à"
```

Comment:
This is the only place we use "carteira". We are using "wallet" in all other places.

Reply:
Fixed.